PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -8,6 +8,7 @@
         'keep *_slimmedMuons_*_*',
         'keep *_slimmedTaus_*_*',
         'keep *_slimmedTausBoosted_*_*',
+        'keep *_selectedCaloJets_*_*',
```

Comment:
to follow the naming convention, please call this slimmedCaloJets
("selected" implies just a selection, not modification that zeroes out values as a part of slimming)